### DISCUSSION TITLE
what i have to do

### DISCUSSION THREAD

**Discussion by KARE-OPEN-SOURCE-SOCEITY** [community/community#69248] [New to GitHub]
how to use this

**Comment by Narla-Venkata-Anand-Sai-Kumar** [comment]
you to upload to projects

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
